apps/core/src/hooks/useTotalFiatBalance.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCurrentAccount } from '@iota/dapp-kit';
+import { useBalance } from './useBalance';
+import { useTokenPrice } from './useTokenPrice';
+import { CoinFormat, formatBalance, useCoinMetadata } from './useFormatCoin';
+import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+
+export function useTotalFiatBalance() {
+    const { data: { price } = {} } = useTokenPrice('iota');
+    const address = useCurrentAccount()?.address;
+    const { data: coinBalance } = useBalance(address!);
```

Comment:
I prefer to do a return if the address does not exist, than to use `address!`, wdyt?
`if (!address) return;`